mysql/errname.go

```diff
@@ -931,7 +931,7 @@ var MySQLErrName = map[uint16]string{
 	ErrMemExceedThreshold:         "%s holds %dB memory, exceeds threshold %dB.%s",
 	ErrForUpdateCantRetry:         "[%d] can not retry select for update statement",
 	ErrAdminCheckTable:            "TiDB admin check table failed.",
-	ErrTxnTooLarge:                "Transaction is too large",
+	ErrTxnTooLarge:                "Transaction is too large, size: %d",
```

Comment:
Good advice, however, it's actually supposed to use ErrDupEntryWithKeyName instead of this error. It's hard to choose here. The Duplicate Entry error code that TiDB has been using all the time seems to be wrong. Change the error code, I am afraid there will be some compatibility problems. This error is too common. I don't know if I should follow the same message as ErrDupEntryWithKeyName. @kennytm